terraform/webhook/main.py

```diff
@@ -76,49 +52,124 @@ def summarize_text(text: str, parameters: None | dict[str, int | float] = None)
     vertexai.init(
         project=_PROJECT_ID,
         location=_LOCATION,
-        credentials=_CREDENTIALS
     )
 
     final_parameters = _DEFAULT_PARAMETERS.copy()
     if parameters:
-      final_parameters.update(parameters)
+        final_parameters.update(parameters)
 
     model = TextGenerationModel.from_pretrained("text-bison@001")
     response = model.predict(
-      f'Provide a summary with about two sentences for the following article: {text}\n'
-      'Summary:'
-    , **final_parameters)
+        f'Provide a summary with about two sentences for the following article: {text}\n'
+        'Summary:',
+        **final_parameters,
+    )
     print(f"Response from Model: {response.text}")
 
     return response.text
 
 
-@functions_framework.cloud_event
-def entrypoint(request):
+def entrypoint(request: object) -> dict[str, str]:
```

Comment:
question: do we no longer need the`functions_framework.cloud_event` decorator?

Reply:
Yup, no longer necessary.